packages/manager/src/components/Button/button.spec.js

```diff
@@ -1,148 +1,166 @@
-            const secondaryButton = $(button.secondary);
-            secondaryButtons = $$(button.secondary);
-
-            expect(secondaryButton.isVisible()).toBe(true);
-            expect(secondaryButtons.length).toBe(1);
-        });
-
-       it('should display buttons with transparent backgrounds', () => {
-           secondaryButtons.forEach(button => {
-               expect(button.getCssProperty('background-color').parsed.hex.includes('#000000')).toBe(true)
-            });
-       });
-
-       it('should have secondary included in class of each button', () => {
-           secondaryButtons.forEach(button => {
-               expect(button.getAttribute('class').includes('Secondary')).toBe(true);
-           });
-       });
+  describe('Types Button Suite', () => {
+    let primaryButtons, secondaryButtons;
+
+    beforeAll(() => {
+      navigateToStory(component, childStories[1]);
+    });
+
+    it('should display primary button', () => {
+      $(button.primary).waitForDisplayed();
+
+      primaryButtons = $$(button.primary);
+      expect(primaryButtons.length)
+        .withContext(`should only be one primary button`)
+        .toBe(1);
+    });
+
+    it('should display primary buttons with white text', () => {
+      primaryButtons.forEach(button => {
+        expect(button.getCSSProperty('color').parsed.hex.includes('#ffffff'))
+          .withContext(`incorrect color`)
+          .toBe(true);
+      });
+    });
+
+    it('should have primary included in the class of each button', () => {
+      primaryButtons.forEach(button => {
+        expect(button.getAttribute('class').includes('Primary'))
+          .withContext(`missing Primary class`)
+          .toBe(true);
+      });
+    });
+
+    it('should display secondary button', () => {
+      const secondaryButton = $(button.secondary);
+      secondaryButtons = $$(button.secondary);
+
+      expect(secondaryButton.isDisplayed())
+        .withContext(`secondary button should be displayed`)
+        .toBe(true);
+      expect(secondaryButtons.length)
+        .withContext(`should only be one secondary button`)
+        .toBe(1);
     });
 
-    describe('Primary Dropdown', () => {
-        let primaryDowndowns;
-
-        beforeAll(() => {
-            navigateToStory(component, childStories[2]);
-            browser.waitForVisible(button.generic);
-        });
-
-        it('should display dropdown buttons with carat', () => {
-            primaryDropdowns = $$(button.primaryDropdown);
-            primaryDropdowns.forEach(d => {
-                expect(d.$('svg').isVisible()).toBe(true);
-            });
-        });
-
-        it('should have primary included in class of each dropdown', () => {
-            primaryDropdowns.forEach(d => {
-                expect(d.getAttribute('class').includes('Primary')).toBe(true);
-            });
-        });
+    it('should display buttons with transparent backgrounds', () => {
```

Comment:
perhaops this test can be called `it('should display secondary buttons with transparent backgrounds')`

Reply:
I am thinking that the button naming may have changed since this test was created but it can still be run.

Changed test name.